public/blogPosts/fall2019/js-chat-1/index.md

```diff
@@ -717,94 +754,94 @@ galen.sayHi();
 // Hi, Tim. 😀
 ```
 
-> Harder Exercise: Can you implement `bind` with `apply` with `call`? 
+> Harder Exercise: Can you implement `bind` with `apply` with `call`?
 
 ### 🍏Application: React callbacks
 
 If you have used React before, you might have used `bind`:
 
 ```jsx
 class MyComponent extends React.Component {
-    constructor() {
-        this.state = { count: 0 };
-        // focus on the following line
-        this.incrementCounter = this.incrementCounter.bind(this);
-    }
-
-    incrementCounter() {
-        const currentCount = this.state.count;
-        this.setState({ count: currentCount + 1 });
-    }
-
-    render() {
-        const { count } = this.state;
-        return (
-            <div>
-                <h1>count: {count}</h1>
-                <button onClick={this.incrementCounter}> plus 1 </button>
-            </div>
-        );
-    }
+	constructor() {
+		this.state = { count: 0 };
+		// focus on the following line
+		this.incrementCounter = this.incrementCounter.bind(this);
+	}
+
+	incrementCounter() {
+		const currentCount = this.state.count;
+		this.setState({ count: currentCount + 1 });
+	}
+
+	render() {
+		const { count } = this.state;
+		return (
+			<div>
+				<h1>count: {count}</h1>
```

Comment:
same nit here: if this extra spacing leads to less readability, revert back to original spacing. Otherwise, feel free to ignore!